low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -166,8 +142,8 @@ def validate(self):
         pass
 
     def process(self):
-        # TODO: Import relevant python script here for further execution.
-        pass
+        from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.sspl_post_install import SSPLPostInstall
```

Comment:
Please keep this import statement at top.

Reply:
We are avoiding make unnecessary imports unless we call the appropriate class.
That's why we are importing here.

Reply:
When we say SSPL is working one, all its modules included.
Keeping the import statement at top will report this state early. 

Our time will be spent more on doing re-setup if sspl_setup fails with import statement which is kept at last.
Its my suggestion only.

Reply:
@mariyappanp these imports are for different setup stages, executed in order independently. So any error encountered (including due to imports) during any setup stage needs to fail independently and exit with non-zero code.